e2e/cypress/integration/ui/infra-proxy/chef-server-details.spec.ts

```diff
@@ -0,0 +1,228 @@
+describe('chef server details', () => {
+  let adminIdToken = '';
+  const now = Cypress.moment().format('MMDDYYhhmm');
+  const cypressPrefix = 'infra';
+  const serverName = `${cypressPrefix} server ${now}`;
+  const updatedServerName = `${cypressPrefix} updated server ${now}`;
+  const serverID = serverName.split(' ').join('-');
+  const customServerID = `${cypressPrefix}-custom-id-${now}`;
+  const serverFQDN = 'chef-server-1617089723092818000.com';
+  const serverIP = '176.119.50.159';
+  const orgName = `${cypressPrefix} org ${now}`;
+  const generatedOrgID = orgName.split(' ').join('-');
+  const customOrgID = `${cypressPrefix}-custom-id-${now}`;
+  const adminUser = 'test_admin_user';
+
+  // using dummy admin key value for creating the org
+  const adminKey = 'Dummy--admin--key';
```

Comment:
while this works, I think it might be better practice to set an RSA key in the env variables, and then use that in any file that needs an RSA moving forward:
see: https://docs.cypress.io/guides/guides/environment-variables#Setting

i.e.:  
```
const ADMIN_RSA_KEY = Cypress.env('RSA_KEY');
```

Reply:
ahh makes sense - thank you!